Tella/Scenes/Settings/Views/Servers/Uwazi/Step2/UwaziServerAccessSelectionView.swift

```diff
@@ -0,0 +1,100 @@
+//
+//  UwaziServerAccessSelectionView.swift
+//  Tella
+//
+//  Created by Robert Shrestha on 4/18/23.
+//  Copyright © 2023 INTERNEWS. All rights reserved.
+//
+
+import SwiftUI
+
+struct UwaziServerAccessSelectionView: View {
+    @State var isButtonValid = true
+    @Environment(\.presentationMode) var presentationMode: Binding<PresentationMode>
+    // TODO: Ask if there is any good way to do it
+    @State var isLoginSelected: Bool = false
```

Comment:
What do you think if you create an enum `UwaziAccessServerType` and save the current selected value to this variable

        enum UwaziAccessServerType {
        case public
        case private
        }

Reply:
@dhekra-rouatbi When the user lands on the view according to the code above the `LOG IN` button will be selected by default so I added a extra case called `none` which is the default value for `accessServerType `.

So the code turns into
`
enum UwaziAccessServerType {
        case publicServer
        case privateServer
        case none
    }
`
` @State var accessServerType: UwaziAccessServerType = .none`